scripts/image

```diff
@@ -332,7 +332,7 @@ fi
       tar cf $TARGET_IMG/$IMAGE_NAME.tar -C target $IMAGE_NAME
 
     # create image files if requested
-      if [ "$1" = "mkimage" -a -n "$BOOTLOADER" ]; then
+      if [ "$1" = "amlpkg" -o "$1" = "mkimage" -a -n "$BOOTLOADER" ]; then
```

Comment:
This test is suspect... whenever `$1 == amlpkg` the condition is `true`, even when `$BOOTLOADER` is blank.

Is this what you meant?
```
if [[ ( "$1" = "amlpkg" || "$1" = "mkimage" ) && -n "$BOOTLOADER" ]]; then
```